h2o-r/h2o-package/R/models.R

```diff
@@ -1469,6 +1469,28 @@ h2o.coef_norm <- function(object) {
   }
 }
 
+grabCoeff <- function(tempTable, nameStart, standardize=FALSE) {
+    coeffNamesPerClass <- tempTable$names # contains coeff names per class
+    totTableLength <- length(tempTable)
+    numClass <- totTableLength
+    startIndex <- 2
+    endIndex <- numClass
```

Comment:
please delete line 1475 numClass <- totTableLength

Please replace line 1477 endIndex <- numClass with the following line:
    endIndex <- (totTableLength-1)/2+1